app/cells/decidim/participatory_documents/suggestion/admin_answer.erb

```diff
@@ -1 +1,4 @@
-<%= translated_attribute(model.answer) %>
+<div class="comment__footer">
+  <%= author_cell(reverse_order: true) %>
```

Comment:
As this is the admin answer. It is unecessary to use the author cell here. Let's just put the official answer text here.
This way we can remove any use of the @reverse_order car and simplify the code of the author cell